docs/writing_plugins/api.rst

```diff
@@ -712,6 +664,62 @@ Example:
     var $buttons = wym.get$Buttons();
 
 *******
+Dialogs
+*******
+
+``dialog(dialogObject)``
```

Comment:
I've altered the documentation here to correspond to that change, as well. Please review.